src/index.html

```diff
@@ -1,13 +1,298 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBiKE</title>
+    <link
+      rel="icon"
+      href="images/favicon.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                class="top-bar__logo"
+                src="images/logo-2x.png"
+                alt="Bike logo"
+              >
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
              <a href="tel:+12345555555" class="icon icon--phone">
```